packages/calypso-codemods/package.json

```
@@ -0,0 +1,29 @@
+{
+	"name": "calypso-codemods",
+	"version": "0.1.6",
+	"description": "jscodeshift transforms used to upgrade calypso code",
+	"main": "./api.js",
+	"bin": {
+		"calypso-codemods": "./index.js"
+	},
+	"scripts": {
+		"test": "jest"
+	},
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/calypso-codemods"
+	},
+	"author": "Jake Fried <samouri@users.noreply.github.com>",
```

Should be

```suggestion
	"author": "Automattic Inc.",
```

per #36534